source/Calamari.Tests/Fixtures/Integration/Packages/DockerImagePackageDownloaderFixture.cs

```diff
@@ -57,8 +57,8 @@ public void PackageWithoutCredentials_Loads()
         [RequiresDockerInstalledAttribute]
         public void DockerHubWithCredentials_Loads()
         {
-            const string privateImage = "octopusdeploy/private-alpine";
-            var version =  new SemanticVersion("1.0.0");
+            const string privateImage = "octopusdeploy/octo-prerelease";
+            var version =  new SemanticVersion("7.3.7");
```

Comment:
I think you mean `7.3.7-alpine`, `7.3.7` does not exist!

Reply:
Could've sworn we had a manifest for that... maybe only for prod builds..
```suggestion
            var version =  new SemanticVersion("7.3.7-alpine");
```